src/objects/cat.js

```diff
@@ -44,9 +55,10 @@ class Cat {
   }
 
   slowDown() {
-    if (this.energy > 0) {
+    if (this.energy > 0 && this.energyGotBurnt()) {
       // remove more energy for a higher energy level:
       this.energy -= Math.floor(this.speed() / 7) || 1;
+      this.setNextSlowDownTime(this.slowDownPeriod);
```

Comment:
@kennethlarsen FYI: I introduced a time based energy removal. 
Since the cat is walking, it burns energy. I liked the idea that you need to keep feeding it. But the loss is way less than if you slow down on each update. We most probably have to adjust the value for a good balance later on anyways, once the mice-feeding is implemented.